scripts/joinmarket-qt.py

```diff
@@ -2003,19 +2021,22 @@ def selectWallet(self, testnet_seed=None):
             #ignore return value as there is no decryption failure possible
             self.loadWalletFromBlockchain(firstarg, pwd)
 
-    def loadWalletFromBlockchain(self, firstarg=None, pwd=None):
+    def loadWalletFromBlockchain(self, firstarg=None, pwd=None, rethrow=False):
```

Comment:
This `rethrow` flag is a temporarily solution to keep the behavior of "Generate Wallet" the same as before, while allowing the error message to be displayed directly in the "Open Wallet Dialog".

Eventually, when we have fully completed our redesign of the open/create wallet workflow, this flag would no longer be necessary and will be removed at that point.